src/components/frame/Extensions/Pages/LazyConsensusPage/LazyConsensusPage.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
```

Comment:
`src/components/frame/Extensions/pages/LazyConsensusPage/LazyConsensusPage.tsx`

Change the directory name to `pages` as we have on the main branch